js/server.js

```diff
@@ -38,7 +38,7 @@ var Server = function(config, callback) {
 			res.status(403).send("This device is not allowed to access your mirror. <br> Please check your config.js or config.js.sample to change this.");
 		});
 	});
-	app.use(helmet());
+	app.use(helmet(config.helmetOptions));
```

Comment:
Well .. after thinking more calm this, we need  keep the same behavior at today. If nothing setting for `helmetOptions`  we not should  be add be default values.

If we set the defaults values there possibility to get more troubles, some external modules can not work anymore ...  and us trying to solve,  apply fixes here and there.

So... if somebody want to include the values by default need a policy or plan for this. First of all, add a warning or notice by log where it said it the exists of the new configuration option in an the future will be apply the default values ...if not this is an open windows to get more troubles.